xcode/Subconscious/Shared/Services/AddressBookService.swift

```diff
@@ -10,6 +10,25 @@ import Combine
 // temp
 import SwiftUI
 
+enum AddressBookError: Error {
+    case cannotFollowYourself
+    case alreadyFollowing
+    case other(String)
+}
+
+extension AddressBookError: LocalizedError {
+    var errorDescription: String? {
+        switch self {
+        case .cannotFollowYourself:
+            return NSLocalizedString("You cannot follow yourself.", comment: "Address Book error description")
```

Comment:
We should use [`String(localized:)`](https://developer.apple.com/documentation/swift/string/init(localized:)) here instead. It's the new Swift-native API for this.